contrib/create-basic-configdrive

```diff
@@ -42,13 +42,14 @@ ssh_authorized_keys:
   - <SSH_KEY>
 hostname: <HOSTNAME>
 "
-REGEX_SSH_FILE="^ssh-(rsa|dss|ed25519) [-A-Za-z0-9+\/]+[=]{0,2} .+"
+REGEX_SSH_FILE="^(ssh-(rsa|dss|ed25519)|ecdsa-sha2-nistp521) [-A-Za-z0-9+\/]+[=]{0,2} .+"
 
-while getopts "d:e:H:i:n:p:S:t:l:u:h" OPTION
+while getopts "d:e:c:H:i:n:p:S:t:l:u:h" OPTION
 do
     case $OPTION in
         d) ETCD_DISCOVERY="$OPTARG" ;;
         e) ETCD_ADDR="$OPTARG" ;;
+        c) CLOUD_CONFIG_FILE="${OPTARG}" ;;
```

Comment:
We should probably also update `USAGE` in the same file, to add the `-c` option.

Reply:
ah yeah, i forgot the usage info. 
and yes, i was not yet going to axe all that stuff.